src/main/java/net/atos/zac/solr/schema/SolrSchemaV7.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Atos
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.solr.schema;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.solr.client.solrj.request.schema.SchemaRequest;
+
+import net.atos.zac.solr.SolrSchemaUpdate;
+import nl.info.zac.search.model.zoekobject.ZoekObjectType;
+
+
+class SolrSchemaV7 implements SolrSchemaUpdate {
```

Comment:
Good to add a JavaDoc comment here explaining why this schema update was created.